src/Client/Resources/keen-io-3_0.php

```diff
@@ -1,5 +1,14 @@
 <?php
 
+$organizationKeyValue = array(
```

Comment:
There is a drawback here: the huge config array is now a dynamic array, meaning that OPCache cannot keep it in shared memory anymore.